src/io/flutter/run/bazel/BazelFields.java

```diff
@@ -222,17 +180,17 @@ GeneralCommandLine getLaunchCommand(@NotNull Project project,
       throw new ExecutionException(e);
     }
 
-    final VirtualFile appDir = getAppDir(project);
```

Comment:
Pulling appDir out because in bazel build systems, we can use the workspace root as the project root.